File: src/main/java/com/example/risk/boundary/api/SearchResult.java
package com.example.risk.boundary.api;

import lombok.Builder;
import lombok.Value;

import java.math.BigDecimal;

@Value
@Builder
public class SearchResult {

    Long id;

    String wkn;
    String name;
    int quantity;
    BigDecimal purchasePrice;
    BigDecimal transactionCosts;
    BigDecimal investment;
    BigDecimal notionalSalesPrice;
    BigDecimal notionalRevenue;
    BigDecimal positionRisk;
}

File: src/main/js/SearchResult.js
import React from "react";

const SearchResult = ({ result }) => (
  <div>
    {result.id}
    <br />
    {result.wkn}
    <br />
    {result.name}
    <br />
    {result.quantity}
    <br />
    {result.purchasePrice}
    <br />
    {result.transactionCosts}
    <br />
    {result.investment}
    <br />
    {result.notionalSalesPrice}
    <br />
    {result.notionalRevenue}
    <br />
    {result.positionRisk}
    <br />
  </div>
);

export default SearchResult;
